File: design-patterns/go/abstract_factory/transport/motorcycle.go
// motorcycle.go

package main

import "fmt"

type Motorcycle struct {
}

func NewMotorcycle() (motorcycle *Motorcycle) {
	motorcycle = &Motorcycle{}
	return
}

func (motorcycle *Motorcycle) Start() {
	fmt.Println("Motorcycle Started")
}

func (motorcycle *Motorcycle) Stop() {
	fmt.Println("Motorcycle Stopped")
}

func (motorcycle *Motorcycle) Repair() {
	fmt.Println("Motorcycle Repair")
}

File: design-patterns/php/abstractfactory/transport/Motorcycle.php
<?php
// Motorcycle.php

namespace BigBoxCode\DesignPattern\AbstractFactory\Transport;

class Motorcycle implements Transport {
    public function start(): void {
        echo "Motorcycle Started\n";
    }

    public function stop(): void {
        echo "Motorcycle Stopped\n";
    }

    public function repair(): void {
        echo "Motorcycle Repair\n";
    }
}